.github/workflows/deployment.yml

```diff
@@ -55,10 +55,15 @@ jobs:
       - name: Run GitVersion
         id: gitversion
         uses: gittools/actions/gitversion/execute@v0.9.4
-      - name: Set up JDK
-        uses: actions/setup-java@v1
+      - name: Setup JDK 15 from jdk.java.net
+        uses: sormuras/download-jdk@v1
         with:
-          java-version: 15
+          feature: 15
+      - uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JDK_VERSION }}
+          jdkFile: ${{ env.JDK_FILE }}
+      - run: java --version
```

Comment:
Remove this (debug statement)?

Reply:
This stems from the MWE https://github.com/sormuras/download-jdk#minimal-example-with-default-feature-and-using-environment-variables. I also thought about its usefulenes. Since there is not the concept of LOGGER.debug and Logger configuration, I would keep it to ensure quick debugging. We will surely have regularly JDK issues the next years.